tests/tests_insights_client_register.yml

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Skip when running with embedded Candlepin
+  meta: end_play
+  when:
+    - lookup('env', 'LSR_RHC_TEST_DATA') | length == 0
+
+- name: Basic insights-client register/unregister
+  hosts: all
+  gather_facts: true
+
+  tasks:
+    - name: Register insights
+      include_role:
+        name: linux-system-roles.rhc
+      vars:
```

Comment:
to register properly, this also needs `rhc_auth`, `rhc_organization`, and `rhc_server` -- pick them as available in `tests_register_unregister.yml`; to make them work, you need to import the file from `LSR_RHC_TEST_DATA` with the config -- add this task before the first register task:
```yaml
    - name: Import test data
      include_vars:
        file: "{{ lookup('env', 'LSR_RHC_TEST_DATA') }}"
      delegate_to: 127.0.0.1
      become: false
```